### ISSUE TITLE
adding eu-west-3 (Paris) for AWS

### ISSUE THREAD

**Issue by ejreymond-christensen** [trailofbits/algo#920]
### OS / Environment (where do you run Algo on)

```
MacOS High Sierra 10.13.4
ansible 2.4.3.0
```

### Cloud Provider (where do you deploy Algo to)

```
AWS - EC2
```

### Summary of the problem

Is it possible to add AWS region: ` eu-west-3 ` for `EU (Paris)`

### Steps to reproduce the behavior

1. run ./algo and follow prompts

### Full log

```
./algo

  What provider would you like to use?
    1. DigitalOcean
    2. Amazon Lightsail
    3. Amazon EC2
    4. Microsoft Azure
    5. Google Compute Engine
    6. Scaleway
    7. OpenStack (DreamCompute optimised)
    8. Install to existing Ubuntu 16.04 server (Advanced)

Enter the number of your desired provider
: 3

Enter your aws_access_key (http://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html)
Note: Make sure to use an IAM user with an acceptable policy attached (see https://github.com/trailofbits/algo/blob/master/docs/deploy-from-ansible.md).
[pasted values will not be displayed]
[AKIA...]: 

Enter your aws_secret_key (http://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html)
[pasted values will not be displayed]
[ABCD...]: 

Name the vpn server:
[algo]: algoFrance


  What region should the server be located in?
    1.   us-east-1           US East (N. Virginia)
    2.   us-east-2           US East (Ohio)
    3.   us-west-1           US West (N. California)
    4.   us-west-2           US West (Oregon)
    5.   ap-south-1          Asia Pacific (Mumbai)
    6.   ap-northeast-2      Asia Pacific (Seoul)
    7.   ap-southeast-1      Asia Pacific (Singapore)
    8.   ap-southeast-2      Asia Pacific (Sydney)
    9.   ap-northeast-1      Asia Pacific (Tokyo)
    10.  eu-central-1        EU (Frankfurt)
    11.  eu-west-1           EU (Ireland)
    12.  eu-west-2           EU (London)
    13.  ca-central-1        Canada (Central)
    14.  sa-east-1           São Paulo
Enter the number of your desired region:
[1]:
```

**Comment by dguido** [comment]
I'm happy to take a PR to address this issue!

**Comment by adamluk** [comment]
This can now be closed as per [ PR #928] (https://github.com/trailofbits/algo/pull/928), which is now merged.
